lib/screens/settings.dart

```diff
@@ -29,34 +29,36 @@ class _SettingsScreenState extends State<SettingsScreen> {
                     child: Image.network(imageUrl,
                         width: 100, height: 100, fit: BoxFit.cover)),
                 SizedBox(height: 20),
-                Text('Welcome, this is the Settings Screen,',
-                    textAlign: TextAlign.center),
                 Text(displayName,
                     textAlign: TextAlign.center,
+                    
                     style:
-                        TextStyle(fontWeight: FontWeight.bold, fontSize: 25)),
+                        TextStyle(fontWeight: FontWeight.bold,  fontSize: 16)),
                 SizedBox(height: 20),
```

Comment:
I believe this is bad practice. Remove it and just add a Padding widget in the Text widget or in the other sized box